ime/app/src/main/res/values/strings.xml

```diff
@@ -552,6 +552,7 @@
     <string name="extension_kbd_extension_numbers_symbols">Numbers and symbols</string>
     <string name="extension_kbd_bottom_minimal">Minimal</string>
     <string name="extension_kbd_bottom_minimal_no_emoji">Minimal (no emoji)</string>
+    <string name="extension_kbd_bottom_swiftkey">SwiftKey</string>
```

Comment:
I'm pretty sure I can not use this trademark in my repo. Can you think of a different description?

Reply:
Hmm, could we use "SwiftKey-style" or "SwiftKey-like"? Or something else entirely?